index.html

```diff
@@ -0,0 +1,163 @@
+          taking place in October, in Korea.
+        </p>
+        <p id="event-time">2015.10.15(THU) – 16(FRI)</p>
+        <p id="event-location">
+          @ National Museum of Korea, Art Center Nabi and more
+        </p>
+      </div>
+    </header>
+    <main>
+      <section id="main-program">
+        <h2>Main Program</h2>
+        <hr />
+        <ul id="program-list">
+          <li>
+            <img src="./images/icons/lecture-icon.png" alt="lecture icon" />
+            <h3>Lecture</h3>
+            <p>
+              Listen to the speakers from various countries about the messeges
+              of sharing and opening
+            </p>
+          </li>
+          <li>
+            <img
+              src="./images/icons/cc-exhibition.png"
+              alt="OS Exhibition icon"
+            />
+            <h3>OS Exhibition</h3>
+            <p>
+              Appreciate various creations applying OS license of artists,
+              organized from Art Center Nabi.
+            </p>
+          </li>
+          <li>
+            <img src="./images/icons/forum_icon.png" alt="Forum icon" />
+            <h3>Forum</h3>
+            <p>
+              Have the time to share your thoughts and opinions with experts for
+              each topic.
+            </p>
+          </li>
+          <li>
+            <img src="./images/icons/workshop-icon.png" alt="Workshop icon" />
+            <h3>Workshop</h3>
+            <p>
+              Try creating your own work using open source license rather than
+              just watching at it.
+            </p>
+          </li>
+          <li>
+            <img src="./images/icons/cc-ignite-icon.png" alt="OS Ignite icon" />
+            <h3>OS Ignite</h3>
+            <p>
+              Get opportunities to network with OS affiliates around the world,
+              also after the summit.
+            </p>
+          </li>
+        </ul>
+        <a href="#" id="main-program-know-more-link">See The Whole Program</a>
+      </section>
+      <section id="partner">
+        <h2>Partner</h2>
+        <hr />
+        <ul id="partner-list">
+          <li>
+            <img src="./images/icons/mozilla-icon.png" alt="mozilla icon" />
+          </li>
+          <li>
+            <img src="./images/icons/google-icon.png" alt="google icon" />
+          </li>
+          <li><img src="./images/icons/naver.png" alt="naver icon" /></li>
+          <li>
+            <img src="./images/icons/daumkakao.png" alt="daumkakao icon" />
+          </li>
+          <li>
+            <img src="./images/icons/airbnb-icon.png" alt="airbnb icon" />
+          </li>
+        </ul>
+      </section>
+    </main>
+    <footer>
+      <div>
+        <img src="./images/icons/footer-logo.png" alt="footer-logo" />
+      </div>
+      <div id="footer-copyright-info">
+        <p>2015 Open Source Korea, Some Rights Reserved</p>
+      </div>
+    </footer>
```

Comment:
- [ ] OPTIONAL
Kindly ensure that the footer contents are horizontally aligned properly:

![image](https://user-images.githubusercontent.com/12745474/169349651-c6206b8e-ee7d-4f2a-810b-ee7331d4c92f.png)